packages/shopify-cart/src/client/actions/cartCreate.ts

```diff
@@ -33,20 +33,25 @@ export default async function cartCreate({
   gqlClient,
   params
 }: CreateCartParams): Promise<void | CartResponse> {
+  let shopifyParams: CartInput = {};
```

Comment:
@mdarrik I'd love your take on these changes, given our discussion in https://github.com/getnacelle/nacelle-js/pull/223/files/e755b61c8eb358dfbccf697bde6e07b385d3c843#r960042889 about subtle ways that this action can go sideways.

Reply:
This does resolve the issue I pointed out there. Namely you're assigning all of the params to `shopifyParams` whether or not the `params.lines` exist. It's got that same issue of doing extra work to make TS happier, but it is a lot less type assertions.

Reply:
Thank you for the insights!